src/main/java/io/phasetwo/keycloak/magic/auth/MagicLinkContinuationAuthenticator.java

```diff
@@ -8,6 +8,7 @@
 
 import io.phasetwo.keycloak.magic.MagicLink;
 import io.phasetwo.keycloak.magic.auth.token.MagicLinkContinuationActionToken;
+import io.phasetwo.keycloak.magic.auth.util.AuthenticatorSharedUtils;
```

Comment:
should be a static import. E.g. `import static io.phasetwo.keycloak.magic.auth.util.AuthenticatorSharedUtils.is;`

Reply:
Added static imports for both of the Authenticator to keep it consistent.